File: guitar/src/app/model/order.ts
export class Order {

    //Table ID
    id?: number = 0;

    //User Data
    userId: number = 0;
    userName?: string = "";
    orderId: number = 0;

    //Product Data
    productId: number = 0;
    productName: string = "";
    imageFile: string = "";
    quantity: number = 0;
    price: string = "0.0";
    Address?: string = ""; 
    //SKU??

    //status
    status: string ="";

    //transaction Data
    transactionId: number = 0;
    transactionDate: string ="";
    totalAmount: string = "";
   // paymentType: string = ""; //not yet implemented in orderlist/database comment for now
}

File: src/main/java/com/rosal/model/Order.java
package com.rosal.model;

import lombok.Data;

import java.util.Date;

@Data
public class Order {
    int id;
    int orderId;
    int userId;
    String userName;

    //Product Data
    int productId;
    String productName;
    String imageFile;
    double quantity;
    double price;
    String status;
    String Address;

    //TRANSACTION DATA
    String transactionId;
    Date transactionDate;
    double totalAmount;
}
